src/app/sidebar/components/Card.tsx

```diff
@@ -0,0 +1,22 @@
+import { Box, Heading } from "@chakra-ui/react";
+import React from "react";
+
+export interface CardProps {
+  title: string;
+  subject: string;
+  code: string;
+  selected?: boolean;
+}
+
+export function Card(props: CardProps): JSX.Element {  
+  return (
+    <Box bgColor={props.selected ? 'orange.200' : 'white'} color="black" m="5" p="4">
```

Comment:
Something really picky, but we should stick to either using ' or "

Reply:
inb4 prettier. we'll fix it in a single go i think